script/int-log-compiler.sh

```diff
@@ -54,6 +54,23 @@ while test $# -gt 0; do
     shift
 done
 
+sanity_test ()
+# Verify the running shell and OS environment is sufficient to run these tests.
+{
+# Check special file
+if [ ! -e /dev/urandom ]; then
```

Comment:
No indentation. See last review comment w/r to coding style.

Reply:
Accepted.  With Solaris source code only tabs and no spaces are allowed for indenting.  I need to get used to using the opposite convention now.